### PR TITLE
update plugin description for QGIS 3.6

### LINKED ISSUE #11: QGIS minimum version
Thanks for the update this weekend about the plugin.

Is the plugin expected to work on QGIS 3.4?
I remember seeing some fixes in QGIS 3.6 and above.

Does the plugin works better with a specific QGIS version?

Related metadata: https://github.com/opengisch/qgis_trackable_project_files/blob/master/trackable_project_files/metadata.txt#L5 where it's written QGIS 3.0.

### PR DESCRIPTION
**Pull request by Gustry**
As I'm not sure if the plugin is useful with QGIS 3.4. AFAIK, my `git diff` is still messy with QGIS 3.4 ;-)

Fix #11

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,3 +6,5 @@
 QGIS project files (.qgs) are XML  files. They tend to change a lot when saving them on different machines. And therefore are hard to track in versioning systems like GIT because the XML attributes are ordered differently every time.
 
 Install this plugin and it will make sure that attributes are ordered alphabetically whenever you save a QGIS file.
+
+The plugin works better if it's used with QGIS ≥ 3.6.
```
Comment (m-kuhn) [inline README.md]:
```suggestion
The plugin works best if it's used with QGIS ≥ 3.6.
```

File: trackable_project_files/metadata.txt [hunk 2]
```diff
@@ -5,6 +5,6 @@ version=dev
 qgisMinimumVersion=3.0
 author=Matthias Kuhn
 email=matthias@opengis.ch
-about=Orders attributes in .qgs project files in a predictable order. This helps tracking QGIS project files in versioning systems like git.
+about=Orders attributes in .qgs project files in a predictable order. This helps tracking QGIS project files in versioning systems like git. The plugin works better with QGIS ≥ 3.6.
```
Comment (m-kuhn) [inline trackable_project_files/metadata.txt]:
```suggestion
about=Orders attributes in .qgs project files in a predictable order. This helps tracking QGIS project files in versioning systems like git. The plugin works best with QGIS ≥ 3.6.
```

**Review by m-kuhn — COMMENTED** [review]
Thanks.
I think you originally proposed "best" which I like better than "better". Hit apply if you agree :)

**Comment by Gustry** [comment]
I was honestly wondering the same thing when I did the commit. But then I followed your comment, because I was trusting you ;-)

**Comment by m-kuhn** [comment]
That's always nice to hear. Just so you know, I trust you too.

**Pull request merged**